### PR TITLE
Cricket fixes and other changes

### PR DESCRIPTION
**Pull request by archit342000**
* Added option for selecting game format - T20 and ODI (Test coming soon).
* Added option for no ball and wide.
* Fixed bug where both teams could bat at the same time.
* Fall of wicket and scoring of runs can take place on the same ball (e.g. run outs).
* Added feature for super overs for t20 matches.
* Fixed issue of no limits on overs or wickets.
* Added feature for displaying result.
* Added updated apk to the release folder.

### CODE REVIEW

**Review by chanmol1999 — CHANGES_REQUESTED** [review]
Add a gif/video portraying the changes

File: Scorecounter/app/src/main/res/layout/activity_select_format.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            android:layout_margin="20dp"/>
+
+        <Button
+            android:id="@+id/ODI"
+            style="@style/button"
+            android:text="ODI"
+            android:textSize="36sp"
+            android:background="@drawable/rounded_button_green"
+            android:padding="8dp"
+            android:layout_margin="20dp"/>
```
Comment (chanmol1999) [inline Scorecounter/app/src/main/res/layout/activity_select_format.xml]: Also, don't use hardcoded values for dimensions.
Use resource files instead

File: Scorecounter/app/src/main/res/layout/activity_first_team.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            android:textSize="36sp"
+            android:background="@drawable/rounded_button_blue"
+            android:padding="8dp"
+            android:layout_margin="20dp"/>
+
+        <Button
+            android:id="@+id/TeamBFirst"
+            style="@style/button"
+            android:text="Team B"
+            android:textSize="36sp"
```
Comment (chanmol1999) [inline Scorecounter/app/src/main/res/layout/activity_first_team.xml]: Same don't use hardcoded values

File: Scorecounter/app/src/main/res/layout/activity_first_team.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_vertical"
+        android:orientation="vertical">
+
+        <TextView
+            style="@style/headings"
+            android:text="Select team to bat first."
```
Comment (chanmol1999) [inline Scorecounter/app/src/main/res/layout/activity_first_team.xml]: Use resource files for strings.

**Review by chanmol1999 — CHANGES_REQUESTED** [review]

**Comment by archit342000** [comment]
ezgif-1-7942adcd33d0

**Comment by archit342000** [comment]
ezgif-1-d52c82836f94

**Comment by archit342000** [comment]
ezgif-1-bcf2e9d5d2b2

File: Scorecounter/app/src/main/res/layout/activity_select_format.xml (hunk 1 above)
Comment (archit342000) [reply Scorecounter/app/src/main/res/layout/activity_select_format.xml]: done

File: Scorecounter/app/src/main/res/layout/activity_first_team.xml (hunk 2 above)
Comment (archit342000) [reply Scorecounter/app/src/main/res/layout/activity_first_team.xml]: done

File: Scorecounter/app/src/main/res/layout/activity_first_team.xml (hunk 3 above)
Comment (archit342000) [reply Scorecounter/app/src/main/res/layout/activity_first_team.xml]: done

**Comment by archit342000** [comment]
Added gifs @chanmol1999

**Review by chanmol1999 — APPROVED** [review]

**Pull request merged**